src/qml/Mysthea/Pages/GameSetup.qml

```diff
@@ -0,0 +1,87 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.4
+import QtQuick.Layouts 1.3
+
+Page {
+    id: root
+    padding: 8
+
+    readonly property string apps: "\uE5C3"
+    readonly property string menu: "\uE5D2"
+    readonly property string navigateBefore: "\uE408"
+    readonly property string navigateNext: "\uE409"
+
+    property alias currentIndex: _swipeView.currentIndex
+
+    header: ToolBar {
+        RowLayout {
+            anchors.fill: parent
+            ToolButton {
+                text: root.apps
```

Comment:
If you want to set an icon as text you must set `font.family` and `font.pixelSize`.